source/src/styles/index.css

```diff
@@ -211,64 +215,80 @@ menu button:hover{
 
 #up-next[data-selected = "true"], #completed[data-selected = "true"] {
     background-color: #f36060;
+    color: white;
+    text-align: center;
+    font-weight: 500;
+    font-size: 5.875vw;
```

Comment:
Having the font-size or padding and stuff be a function of viewport width is a bad idea. 
It makes it hard to track every property for every media query - you miss one thing and the UI can go awry.

The original design of having `font-size: var(--main-font-size);` and then just changing the CSS variables in the media query is good design IMHO.